apps/core/urls.py

```diff
@@ -4,9 +4,12 @@
 from rest_framework import permissions
 
 from .views import (
-    DecoratedTokenObtainPairView, DecoratedTokenBlacklistView, DecoratedTokenVerifyView,
-    DecoratedTokenRefreshView, RegisterView, VerifyEmailView, ReverifyEmailView,
-    PasswordResetView, PasswordResetConfirmView, PasswordChangeView,
+    DecoratedTokenObtainPairView, DecoratedTokenBlacklistView,
+    DecoratedTokenVerifyView,
+    DecoratedTokenRefreshView, RegisterView, VerifyEmailView,
+    ReverifyEmailView,
+    PasswordResetView, PasswordResetConfirmView, PasswordResetChangeView,
+    PasswordChangeView,
 )
```

Comment:
In general, there is no official recommendation on how to format multi-line imports in Python.
But I recommend using one name on one line, ex
```
from .views import (
    DecoratedTokenObtainPairView,
    DecoratedTokenBlacklistView,
    DecoratedTokenVerifyView,
    ...
)
```

Or replace it with `import views` and use in file like `views.DecoratedTokenObtainPairView`